hologram_dashboard/extension/.gitignore

```diff
@@ -0,0 +1,46 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
```

Comment:
which part of this file is changed?

Reply:
added hologram_dashboard/node_modules and
hologram_dashboard/package-lock.json